src/Ringo.Bot/services/healthService.ts

```diff
@@ -9,48 +16,82 @@ export async function getHealth(req, res, next) {
         Status: 'Up',
         Version: packageJson.version,
         DateTime: new Date(),
-        Services:[]
+        Services: []
     }
 
+    try {
```

Comment:
Promise.all() is pretty handy here. I'm calling getTableStorageHealth twice for demonstration purposes. Will add getGraphDbHealth soon.